opencti-platform/opencti-front/src/private/components/settings/attributes/VocabularyPopover.tsx

```diff
@@ -96,18 +95,19 @@ const VocabularyPopover: FunctionComponent<VocabularyPopoverProps> = ({ vocab, p
   } = useDeletion({ handleClose });
   const [commit] = useMutation(VocabularyPopoverDeletionMutation);
 
-  const submitDelete = () => {
+  const submitDelete = (vocabulary: useVocabularyCategory_Vocabularynode$data) => {
```

Comment:
It's supposed to work with the vocab property no ?
It seems strange to me to have to pass this property via this method

Reply:
no it doesn't work, since vocab takes the value of an other vocab of the list once we've deleted the current one...

Reply:
the bug comes from that point

Reply:
you're right !

Reply:
fixed